bitnami/appsmith/templates/backend/deployment.yaml

```diff
@@ -215,8 +215,7 @@ spec:
           livenessProbe: {{- include "common.tplvalues.render" (dict "value" .Values.backend.customLivenessProbe "context" $) | nindent 12 }}
           {{- else if .Values.backend.livenessProbe.enabled }}
           livenessProbe: {{- include "common.tplvalues.render" (dict "value" (omit .Values.backend.livenessProbe "enabled") "context" $) | nindent 12 }}
-            httpGet:
-              path: /api/v1/users/me
```

Comment:
Please note there is a `/api/v1/health` endpoint that also checks DB connections. This could be useful for readiness